lib/serve.js

```diff
@@ -37,9 +37,16 @@ function startServer (options) {
   const server = new Hapi.Server()
   server.connection({ port: options.port })
 
+  let cors = true // development only, not recommended in production
+  if (options.cors && options.cors.origins) {
+    cors = {
+      origin: options.cors.origins,
```

Comment:
We have options here, either copy the single origin that they want to use for production, or use the presence of a single origin to decide to use a "*". If they've got no origins, then our local server probably shouldn't either.